data/_models/shared/Federated data access.csv

```diff
@@ -0,0 +1,11 @@
+tableName,tableExtends,tableType,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,refLabel,defaultValue,validation,visible,computed,semantics,profiles,label,description
+Federated data access,,,,,,,,,,,,,,,,,"DataCatalogueFlat,FederatedDataAccess",,List of Armadillo and/or Opal server(s) available for this resource.
+Federated data access,,,resource,ref,1,TRUE,,Resources,,,,,,,,,"DataCatalogueFlat,FederatedDataAccess",,Federated data access defined for this resource
+Federated data access,,,name,ref,1,TRUE,,Agent,,,,,,,,,"DataCatalogueFlat,FederatedDataAccess",,For example: Armadillo or Opal.
```

Comment:
Can you make a separate ontology for this and not use Agent? Something like 'DataSHIELD servers'. Agent will be merged with Organisations, and Opal and Armadillo are definitely not organisations.

Reply:
Reference to Agent removed, added ontology lists.